op-validator/pkg/validations/codes.go

```diff
@@ -0,0 +1,136 @@
+package validations
+
+var descriptions = map[string]string{
+	// SuperchainConfig validations
+	"SPRCFG-10": "SuperchainConfig version mismatch",
```

Comment:
Out of curiosity: is there a specific reference to all of these validation codes?

Reply:
The implementation is the reference